frontend/src/containers/Collection/CollectionList/CollectionList.js

```diff
@@ -2,33 +2,17 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
 
-import { CollectionCard } from "../../../components";
+import { CreateNewCollectionModal, CollectionCard } from "../../../components";
 import { collectionActions, authActions } from "../../../store/actions";
-// import { collectionStatus } from "../../../constants/constants";
 
 import "./CollectionList.css";
 
 class CollectionList extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            // makeNewCollectionStatus: collectionStatus.NONE,
-            collections: [],
-            me: null,
-        };
-    }
-
     componentDidMount() {
         this.props.onGetMe()
```

Comment:
Don't call onGetMe.

You can improve codes like this.

```
    componentDidMount() {
        if (this.props.me) {
            this.props.onGetCollections({ id: this.props.me.id });
        }
    }

    /* eslint-disable react/no-did-update-set-state */
    // It's OK to use setState if it is wrapped in a condition
    // please refer to https://reactjs.org/docs/react-component.html#componentdidupdate
    componentDidUpdate(prevProps) {
        if (this.props.me !== prevProps.me) {
            this.props.onGetCollections({ id: this.props.me.id });
        }
    }
    /* eslint-enable react/no-did-update-set-state */
```

Reply:
The same code is in AccountSetting.js. Please review other codes carefully!

Reply:
Change reflected.